src/gui/src/bufferTreeDescriptor.cpp

```diff
@@ -86,35 +86,35 @@ void BufferTree::populate(odb::dbNet* net)
 
     for (auto* next_iterm : inst->getITerms()) {
       if (!next_iterm->getSigType().isSupply()) {
-        populate(next_iterm->getNet());
+        populate(next_iterm->getNet(), timingSource);
       }
     }
   }
 }
 
-bool BufferTree::isAggregate(odb::dbNet* net)
+bool BufferTree::isAggregate(odb::dbNet* net, bool timingSource)
 {
   if (net == nullptr) {
     return false;
   }
 
   for (auto* iterm : net->getITerms()) {
-    if (BufferTree::isAggregate(iterm->getInst())) {
+    if (BufferTree::isAggregate(iterm->getInst(), timingSource)) {
       return true;
     }
   }
 
   return false;
 }
 
-bool BufferTree::isAggregate(odb::dbInst* inst)
+bool BufferTree::isAggregate(odb::dbInst* inst, bool timingSource)
```

Comment:
why is `timingSource` needed?

Reply:
This function consider buffer only if the TimingSource of the inst is TIMING, but I found that only inst that insert from ECO ( using RSZ ) are getting some kind of TimingSource value.
So the buffer tree don't found most of the buffers in that case, so I added some work around that issue, because Im not fully understood why there was from start the TimingSource check

Reply:
@gadfort I'm also a bit unclear on this check

Reply:
@maliberty I believe when we first added this, we decided that the tree should terminate on buffers and inverters inserted by synthesis (I'm not 100% if that really matters), so unless there is a compelling reason to do so, I would be happy to remove that limit

Reply:
We remove all the synthesis buffers so its only really inverters which wouldn't pass isBuffer() afaik.  I think we can remove it.  

That said I'm surprised it made a difference unless you were looking at a clock net.

Reply:
I encounter that because I created test case with multiple place_cell command manually and then the buffer didn't added to the tree.
My thought was that it should be added to tree in case someone will insert buffer manually.

Reply:
That makes sense.  Go ahead and remove the check